index.html

```diff
@@ -43,6 +36,174 @@ <h1 class="h1-green">
       <i class="fab fa-github"></i>
       <i class="fab fa-angellist"></i>
     </nav>
+    <nav>
+      <img src="images/Action.svg" alt="Action icon">
+    </nav>
```

Comment:
- [ ] [Required] For semantic reasons, I think the `nav` element should be reserved for the navigation bar. Since your navigation bar is at line 18, it would be unnecessary to have multiple nave elements on the page. Please use any other element to wrap other items on your page other than the navigation bar.